node/middlewares/errors.ts

```diff
@@ -0,0 +1,23 @@
+import { TooManyRequestsError } from '@vtex/api'
+import { any } from 'ramda'
+
+const ERROR_429 = 'E_HTTP_429'
+
+const isTooManyRequestError = (error: any) => {
```

Comment:
I think there's an 's' missing in isTooManyRequestError -> isTooManyRequestsError.